introduction_to_amazon_algorithms/forecasting_services_comparison/forecast_example.ipynb

```diff
@@ -0,0 +1,755 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    while True:
</pre><p>Again try to avoid <code>while True</code> ﻿in favor of more specific logic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>